data/routes/scripts/standard_ot_api.lua

```diff
@@ -0,0 +1,60 @@
+local json = nil
+
+local function getPlayersOnline(session)
+	session.responder:setResponseStatus(200, 'OK')
+	session.responder:setResponseBody(json.encode(Game.getPlayers()))
+end
+
+local function getServerStatus(session)
+	local data = {
+		uptime = getWorldUpTime(),
+		ip = configManager.getString(configKeys.SERVER_IP),
+		serverName = configManager.getString(configKeys.SERVER_NAME),
+		port = configManager.getNumber(configKeys.LOGIN_PORT),
+		location = configManager.getString(configKeys.LOCATION),
+		url = configManager.getString(configKeys.URL),
+		server = 'The Forgotten Server',
+		version = '1.3',
+		client = '1098',
```

Comment:
Unless I completely suck at grepping, I don't think so. server, version and clients are the values from https://github.com/otland/forgottenserver/blob/master/src/definitions.h